Test_Automation_Framework/EPAM_TestAutomation_Tests/Basic2_EpamSiteNavigateTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Epam_TestAutomation_BusinessLogic.PageObjects.Pages;
+using Epam_TestAutomation_Core.Browser;
+using NUnit.Framework;
+using OpenQA.Selenium;
+
+namespace Epam_TestAutomation_Tests
+{
+    [TestFixture]
+    public class Basic2_EpamSiteNavigateTests : BaseTest
+    {
+        private SearchResultsPage _searchResultsPage;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _searchResultsPage = new SearchResultsPage();
```

Comment:
Why do you use SearchResultsPage if you navigate to the MainPage?

Reply:
Followed previous comment

Reply:
Yes, but as I can see your browser navigates to the epam.com site and then makes a search from first page and the navigates to the search results page (also for clicking to the careerButton). I mean you should create the separate classes for each page.